app/src/main/java/com/example/newskeletonapp/ui/view/DetailsFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.newskeletonapp.ui.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import com.example.newskeletonapp.R
+import com.example.newskeletonapp.ui.model.DetailsViewModel
+
+class DetailsFragment : Fragment() {
+
+    companion object {
+        fun newInstance() = DetailsFragment()
+    }
+
+    private lateinit var viewModel: DetailsViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return inflater.inflate(R.layout.details_fragment, container, false)
+    }
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        viewModel = ViewModelProvider(this).get(DetailsViewModel::class.java)
```

Comment:
Itt lehetne `by viewModels()`-t használni, nem? :D

Reply:
O dehogynem, de ez valamelyik kesobbi commitban ki is esik szerintem, ha mashol nem, akkor ott ahol bejon a BaseFragment, legalabbis remelem 😁